setup.py

```diff
@@ -6,561 +6,58 @@
-                print("Created %s" % final_package_path)
-
-            for arch in ('osx-64', 'linux-32', 'win-32', 'win-64'):
-                if arch == build_arch:
-                    continue
-                converted_output_dir = os.path.join(python_scoped_package_dir)
-                converted_package_path = os.path.join(converted_output_dir, arch, os.path.basename(package_path))
-                if os.path.isfile(converted_package_path):
-                    print("Already converted to %s from %s for python %s" % (arch, build_arch, python_version))
-                else:
-                    print("Creating %s by conversion %s=>%s" % (converted_package_path, build_arch, arch))
-                    _safe_makedirs(converted_output_dir)
-                    # this automatically creates the "arch" directory to put the package in
-                    code = subprocess.call(['conda', 'convert', '--platform', arch, final_package_path, '--output-dir',
-                                            converted_output_dir])
-                    if code != 0:
-                        raise Exception("Failed to convert from %s to %s to create %s" %
-                                        (build_arch, arch, converted_package_path))
-                    all_final_package_paths.append(converted_package_path)
-
-        print("Packages in " + self.packages_dir)
+def get_description():
+    """Get long description."""
+    with open(os.path.join(HERE, 'README.md'), 'r') as f:
+        data = f.read()
+    return data
 
 
 setup(name='anaconda-project',
-      version=VERSION,
-      author="Continuum Analytics",
-      author_email='info@continuum.io',
+      version=get_version(),
+      keywords=["conda anaconda project reproducible data science"],
       url='http://github.com/Anaconda-Server/anaconda-project',
-      description='Library to load and manipulate project directories',
       license='New BSD',
+      author="Anaconda, Inc",
+      author_email='info@anaconda.com',
+      maintainer='Anaconda, Inc',
+      maintainer_email='info@anaconda.com',
+      description='Library to load and manipulate project directories',
+      long_description=get_description(),
       zip_safe=False,
-      install_requires=REQUIRES,
-      tests_require=TEST_REQUIRES,
-      cmdclass=dict(test=AllTestsCommand,
-                    conda_package=CondaPackageCommand,
-                    version_module=VersionModuleCommand),
-      scripts=[
-          'bin/anaconda-project'
-      ],
-      packages=[
-          'anaconda_project', 'anaconda_project.internal', 'anaconda_project.internal.cli',
-          'anaconda_project.requirements_registry', 'anaconda_project.requirements_registry.providers',
-          'anaconda_project.requirements_registry.requirements'
+      install_requires=REQUIREMENTS,
+      entry_points={
+          'console_scripts': [
+              'anaconda-project = anaconda_project.cli:main',
```

Comment:
good catch

Reply:
>This PR also impacts docs (think mostly CONTIBUTING but maybe also README). Have you reviewed/checked them?

Ok let me update the README, add some badges, update the docs, and update a RELEASE.md file. At the moment the release process will involve updating the version manually, in a subsequent PR I will automate this, but it was just too much for a single PR, ok? The RELEASE.md will reflect this manual process until we update it.


>I'm +1 on merging this after we agree on the open question I had and we fix docs. +0.5 on merging without doc changes and amending right after on a follow up PR.

Make sense, will work on this now.

Reply:
Sounds good. Thanks